### PR TITLE
Support for treat_missing_data attribute in monitor

### PR DESCRIPTION
**Pull request by pickgr**

### CODE REVIEW

File: policy/variables.tf [hunk 1]
```diff
@@ -78,6 +78,11 @@ variable "threshold" {
   description = "The value against which the specified statistic is compared."
 }
 
+variable "treat_missing_data" {
+  type        = "string"
+  description = "See http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html#alarms-and-missing-data"
```
Comment (disaac) [inline policy/variables.tf]: Looks like this value isn't available in the terraform documentation for this variable. [aws_cloudwatch_metric_alarm](https://www.terraform.io/docs/providers/aws/r/cloudwatch_metric_alarm.html#treat_missing_data)

Comment (pickgr) [reply policy/variables.tf]: That's what happens when I try to rely on my memory :)

**Comment by pickgr** [comment]
@disaac - can you re-review please?  One possible improvement I was thinking about was adding a default value of 'missing' in variables.tf to make it a bit more backwards-compatible.  Not sure if that would be a good idea, since that's not really what you should use with CPUUtilization anyway.

**Comment by disaac** [comment]
Added suggested changes to https://github.com/unifio/terraform-aws-asg/tree/dbi-suggestions tried to push to your repo but didn't have permission to push a branch there. Please see branch for changes to apply.

**Comment by pickgr** [comment]
Superceeded by https://github.com/unifio/terraform-aws-asg/pull/24

**Pull request closed without merging**